docs/eigenda/integrations-guides/dispersal/dispersal-api/overview.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 1
+---
+
+# Overview
+
+The EigenDA Disperser is a permissioned, untrusted service hosted by Eigen Labs which serves as a coordinator between clients, the operator set, and onchain Ethereum contracts for the purpose of dispersing __and retriving__ blobs. In the future the EigenDA disperser will be made permissionless, so that clients can interact with the EigenDA network and contracts directly to avoid a liveness dependency on the Eigen Labs disperser.
+
+The source of truth for the Disperser API spec is [disperser.proto](https://github.com/Layr-Labs/eigenda/blob/8ec570b8c2b266fad20ea0af14f0f5d84906c39c/api/proto/disperser/disperser.proto), adjusted to the current release. The goal of this document is to explain this spec at a higher level.
+
+<!-- TODO: Update to point to master, not a specific commit -->
+
+Eigen Labs hosts one disperser endpoint for each EigenDA network. These endpoints are documented in respective network pages [indexed here](../../../networks/README.md).
+
+The EigenDA Disperser exposes 4 endpoints:
+
+1. `DisperseBlob()`
+2. `DisperseBlobAuthenticated()`
+3. `GetBlobStatus()`
+4. `RetrieveBlob()`
+
+These endpoints enable the blob lifecycle, from enqueuing blobs for dispersal to waiting for their dispersal finalization and finally to retrieving blobs from the EigenDA network. The following flowchart describes how move blobs through this lifecycle with respect to these endpoints:
+
+<!-- Flow chart pseudocode -->
+
+```mermaid
+graph EigenDA Disperser API Usage Flow;
+    A[Blob Ready]
+    B[Blob Queued for Dispersal]
+    C[Blob Dispersal Finalized]
+    D[Blob Expired]
+
+    A --> |DisperseBlob()| B
+    A --> |DisperseBlobAuthenticated()| B
+    B -->|GetBlobStatus() != finalized| B;
+    B -->|GetBlobStatus() == finalized| C;
+    C -->|RetrieveBlob()| C;
+    C -->|15 days elapses since dispersal finalization| D;
+```
+
+The Disperser offers an asynchrounous API for dispersing blobs, where clients should poll the GetBlobStatus() endpoint with the dispersal request ID they received from calling one of the two disperse endpoints until the disperser reports the blob as successfully dispersed and finalized.
+
+## Endpoints
+
+### Dispersal Endpoints
+
+There are two dispersal endpoints, `DisperseBlob()` and `DisperseBlobAuthenticated()`, which achive the same thing under different authentication strategies ( counterintuitively, both endpoints are authenticated in mainnet). Both endpoints accept a [DisperseBlobRequest](https://github.com/Layr-Labs/eigenda/blob/4d19b0597d1151a402e328430a0ff5345a831fb6/api/proto/disperser/disperser.proto#L72) message, with the following fields:
+
+:::warning
+It's important to note that during dispersal, blobs are automatically padded with zero bytes such that the final length of the blob in bytes is a multiple of 32. One implication of this is that a naive EigenDA client implementation will not retrieve the exact same blob that was dispersed.
```

Comment:
This warning seems like it requires additional context. I think we should
1. Cover this in the blob serialization requirements page
2. Just reference that page here

Reply:
This seems like a gotcha that we want to surface a high level wherever possible. i.e. "when you disperse a blob that is of a length that is not a multiple of 32 it is end-padded with zero bytes until it is. Read more about it [here]"

Reply:
How's this?

```
Blobs must be encoded in a special way to be compatible with BN254 KZG commitments, and blob dispersal will fail validation if this encoding is not applied. Also important to note that blobs that are not of a byte length that is a multiple of 32 are end-padded with zero bytes on the server side to the next multiple of 32. For more detail see [Blob Serialization Requirements](../blob-serialization-requirements.md).
```